roles/confluent.common/tasks/rbac_setup.yml

```diff
@@ -18,6 +18,7 @@
     kafka_cluster_id: "{{ cluster_id_query.json.data[0].cluster_id }}"
   when: cluster_id_source | default('erp') == 'erp'
 
+# TODO this will not work for Secure ZK or Archive installation method
 - name: Get Kafka Cluster ID from Zookeeper
   shell: /usr/bin/zookeeper-shell localhost:2181 get /cluster/id | grep version
```

Comment:
will this path to zk-shell be changed as part of this PR?

Reply:
No, leaving this TODO for later. Its def wrong, but this code wont run now during upgrades. It would be smarter just to grep kafka's meta.properties file for the clusterId rather than hitting ZK because if ZK has TLS on things are annoyingly complicated.

Reply:
Also I just confirmed this is dead code now, all invocations of the `rbac_setup.yml` task files say to "get cluster id" from ERP now. I'd prefer to leave it as dead code though in the `-post` branch. But I can understand the case that it should be removed. WDYT?

And if the "when rbac is not used/mds not available" question is good, but essentially the only time we call the rbac_setup tasks we assume rbac is on AND the mds is available bc that why this task file is called.

Reply:
Good point. Yes, I agree.